app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,7 @@
     <application
         android:name="com.duckduckgo.app.global.DuckDuckGoApplication"
         android:allowBackup="false"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="${appIcon}"
```

Comment:
We can use manifest placeholders to change the app icon based on the build version. This is done to replace the need of maintaining a separate folder in the source.